### PR TITLE
Improve Generator and update LibUsb

### PR DESCRIPTION
**Pull request by mattico**
See the commit messages.

With these changes I was able to build LibUsbDotNet on Windows. cc #126

### CODE REVIEW

**Comment by mattico** [comment]
I suggest disabling display of whitespace changes in your diff viewer.

File: src/LibUsbDotNet/Generated/ControlSetup.cs [hunk 1]
```diff
@@ -34,37 +34,34 @@
 
 namespace LibUsbDotNet
 {
-    /// <summary>
-    ///  Setup packet for control transfers.
-    /// </summary>
```
Comment (mattico) [inline src/LibUsbDotNet/Generated/ControlSetup.cs:39]: In the newer `libusb.h` there are preprocessor directives between the doc comment and the struct definition, which I believe is what's causing this to get dropped.

**Comment by mcuee** [comment]
@mattico 

The main branch and v2 branch both can be built under Windows now. Do you still think this PR is necessary?

**Comment by mattico** [comment]
I'm not sure, but I don't use this anymore anyway.

**Pull request closed without merging**